### DISCUSSION TITLE
[Docs] Discord Link

### DISCUSSION THREAD

**Discussion by itsalimanuel** [element-plus/element-plus#11331] [General]
Feature type: **`Docs`**

## Details

- Enhanced component name: `N/A`

## Description

The Discord link in the documentation is not working [(check)](https://github.com/element-plus/element-plus/blob/dev/.github/CONTRIBUTING.en-US.md) , update it to https://discord.com/invite/eajuBjGtA4

### Additional comments

_(empty)_

**Comment by ryuhangyeong** [comment]
thank! This has been taken care of!
